setup/coder-for-docker/cloud.md

```diff
@@ -0,0 +1,8 @@
+---
+title: "Cloud deployment"
+description: Learn how to run Coder with Docker in the cloud.
+---
+
+
+
+Forthcoming.
```

Comment:
What should I put here?

Reply:
I think this is where @bpmct's stuff will live, but we might hold off on releasing this.